node/libs/roles/src/proto/attester.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
+
+package zksync.roles.attester;
+
+import "zksync/std.proto";
+
+message L1Batch {
+  optional uint64 number = 1; // required
+  optional std.Timestamp timestamp = 2; // required
+}
+
+message L1BatchQC {
+  optional L1Batch msg = 1; // required
+  optional std.BitVector signers = 2; // required
+  optional AggregateSignature sig = 3; // required
+}
+
+message Msg {
+  oneof t { // required
+    L1Batch l1_batch = 4;
+  }
+}
+
+message SignedBatch {
```

Comment:
SignedMsg

Reply:
Should we modify the `SignedBatchMsg` struct as well? I named it that way to make it clear that the only message the attesters could sign would be an `L1Batch`, but perhaps we can use `Signed` just like the validators and document it accordingly.